lib/Geo/ShapeContainer.cpp

```diff
@@ -48,8 +48,8 @@ static S2Polygon toPolygon(S2LatLngRect const& rect) {
   std::array<S2Point, 4> v{
       rect.GetVertex(0).ToPoint(), rect.GetVertex(1).ToPoint(),
       rect.GetVertex(2).ToPoint(), rect.GetVertex(3).ToPoint()};
-  auto loop = std::make_unique<S2Loop>(v, S2Debug::DISABLE);
-  return S2Polygon{std::move(loop), S2Debug::DISABLE};
+  auto loop = std::make_unique<S2Loop>(v);
```

Comment:
why not `S2Debug::DISABLE`?

Reply:
It's same for us. But I will add it to avoid confusing.
Forgot here